src/display/annotation_layer.js

```diff
@@ -469,11 +471,12 @@ var PopupElement = (function PopupElementClosure() {
   var BACKGROUND_ENLIGHT = 0.7;
 
   function PopupElement(parameters) {
-    this.parentContainer = parameters.parentContainer;
-    this.parentTrigger = parameters.parentTrigger;
+    this.container = parameters.container;
+    this.trigger = parameters.trigger;
     this.color = parameters.color;
     this.title = parameters.title;
     this.contents = parameters.contents;
+    this.hideContainer = parameters.hideContainer;
```

Comment:
Since the case of a Text annotation *without* an associated Popup annotation is special, it would seem a bit more logical if the the parameter was such that the caller *explicitly* had to set it to `true`, in order to get the special case (`this.hideElement = wrapper;` in the current code).